converters/toZigbee.js

```diff
@@ -2364,6 +2364,86 @@ const converters = {
             await entity.read('haElectricalMeasurement', [0xf000, 0xf001, 0xf002]);
         },
     },
+    tuya_led_control: {
+        key: ['color'],
+        convertSet: async (entity, key, value, meta) => {
+            // transtime is ignored
+            const payload = {
+                transtime: 0,
+                hue: Math.round((meta.state.color.h * 254) / 360),
+                saturation: Math.round(meta.state.color.s * 2.54),
+                brightness: meta.state.brightness || 255,
+            };
+            if (value.h) {
+                payload.hue = Math.round((value.h * 254) / 360);
+            }
+            if (value.s) {
+                payload.saturation = Math.round(value.s * 2.54);
+            }
+            if (value.b) {
+                payload.brightness = value.b;
+            }
+            if (value.brightness) {
+                payload.brightness = value.brightness;
+            }
+            if (typeof value === 'number') {
```

Comment:
If I understand correctly, this is never triggered because `brightness` is not in this list: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1661/files#diff-a610ac853a94f2ed75dcf27fa22fb1cf9790749f9a0876e22c087a24e00885c4R2368

For the sake of simplicity I propose to merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/1661/files#diff-a610ac853a94f2ed75dcf27fa22fb1cf9790749f9a0876e22c087a24e00885c4R2404 with this converter.